simulator/src/main/java/com/google/cloud/JSONOutputConsumer.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud;
+
+import com.google.cloud.orderbook.model.MarketDepth;
+import com.google.cloud.orderbook.model.MarketDepth.PriceQuantity;
+import com.google.cloud.orderbook.model.OrderBookEvent;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JSONOutputConsumer implements EventConsumer {
```

Comment:
Perhaps using a standard Json library would have resulted in a shorter code. But it's small enough and we can leave it as-is.

Reply:
Quite possibly. I just wanted to get some JSON output working ASAP. Using Protobuf to JSON routines might have been best.